### PR TITLE
Order management

### PR DESCRIPTION
**Pull request by 5t4mp5**
Single Order form now loads order, user and line items.
Can update status from single order page.

### CODE REVIEW

File: src/OrderForm.js [hunk 1]
```diff
@@ -29,26 +23,25 @@ class OrderForm extends Component{
...
-            orderNumber: o.orderNumber,
-            totalAmount: o.totalAmount*1,
+            orderNumber: this.props.order.orderNumber,
             status: o.status
         }
-        this.props.updateOrder(o.id, final)
-        .then(()=>this.setState({...this.state, errors: []}))
-        .catch(e=>this.setState({...this.state, errors: e.response.data.errors}));
+        this.props.updateOrder(this.props.order.id, final)
+            .then(()=>this.setState({...this.state, errors: []}))
```
Comment (chem2016) [inline src/OrderForm.js:30]: Do we need to set errors : [], since the state is default to have errors : []

File: src/SingleOrderForm.js [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+    componentDidMount() {
+        this.props.getSingleOrderItemsUsers(this.props.user.id, this.props.match.params.orderId)
+            .then(response => response.data)
+            .then(() => this.setState({ status: this.props.order.status, errors: [] }))
+            .catch(e => console.log(e.message))
+    }
+
+    componentDidUpdate(prevProps) {
+        if(JSON.stringify(prevProps) !== JSON.stringify(this.props)){
+            this.props.getSingleOrderItemsUsers(this.props.user.id, this.props.match.params.orderId)
```
Comment (chem2016) [inline src/SingleOrderForm.js:25]: I am seeing a 500 error for get /api/orders/user/undefined, when click on save or submit on single order form. Maybe I need to sync with the latest dev code

**Review by chem2016 — COMMENTED** [review]
Looks good, I only have two comments.

File: src/OrderForm.js (hunk 1 above)
Comment (5t4mp5) [reply src/OrderForm.js:30]: If there were previous errors updating then yes, we would need to clear the errors on success as we do not leave this page.

**Review by 5t4mp5 — COMMENTED** [review]
You are going to get those errors often on did mount, resolves when did update. Not a showstopper

**Pull request merged**